selfdrive/controls/lib/pathplanner.py

```diff
@@ -119,14 +119,10 @@ def update(self, sm, pm, CP, VM):
         if not one_blinker or below_lane_change_speed:
           self.lane_change_state = LaneChangeState.off
         elif torque_applied:
-          self.lane_change_state = LaneChangeState.laneChangeStarting
-
-      # starting
-      elif self.lane_change_state == LaneChangeState.laneChangeStarting and lane_change_prob > 0.5:
-        self.lane_change_state = LaneChangeState.laneChangeFinishing
+          self.lane_change_state = LaneChangeState.laneChangeFinishing
```

Comment:
laneChangeStarting is gone now. Can you remove it from the dict at the top of the file?

Reply:
done